std/io/readers.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 type Reader = Deno.Reader;
 import { encode } from "../encoding/utf8.ts";
+export * from "./limited_reader.ts";
```

Comment:
probably better to just move LimitedReader into this module

Reply:
I wanted to, but was not sure about the reference to the go module / license notice

Reply:
You can just put it at the top of the file... I think everything in here is ported from go